layout/pages/settings/gameplay.xml

```diff
@@ -137,25 +137,37 @@
 					<Label id="colorize3" text="#Settings_Safeguards_Modes2" value="2" />
 				</SettingsEnumDropDown>
 
+				<SettingsEnumDropDown text="#Settings_Safeguards_Respawn" convar="mom_run_safeguard_respawn" tags="respawn">
+					<Label id="colorize1" text="#Settings_Safeguards_ModesNone" value="0" />
+					<Label id="colorize2" text="#Settings_Safeguards_Modes1" value="1" />
+					<Label id="colorize3" text="#Settings_Safeguards_Modes2" value="2" />
+				</SettingsEnumDropDown>
+
 				<SettingsEnumDropDown text="#Settings_Safeguards_RestartMap" convar="mom_run_safeguard_restart" tags="restart">
 					<Label id="colorize1" text="#Settings_Safeguards_ModesNone" value="0" />
 					<Label id="colorize2" text="#Settings_Safeguards_Modes1" value="1" />
 					<Label id="colorize3" text="#Settings_Safeguards_Modes2" value="2" />
 				</SettingsEnumDropDown>
 
-				<SettingsEnumDropDown text="#Settings_Safeguards_SavestateTele" convar="mom_run_safeguard_saveloc_tele" tags="teleport">
+				<SettingsEnumDropDown text="#Settings_Safeguards_RestartStage" convar="mom_run_safeguard_restart_stage" tags="stage">
 					<Label id="colorize1" text="#Settings_Safeguards_ModesNone" value="0" />
 					<Label id="colorize2" text="#Settings_Safeguards_Modes1" value="1" />
 					<Label id="colorize3" text="#Settings_Safeguards_Modes2" value="2" />
 				</SettingsEnumDropDown>
 
-				<SettingsEnumDropDown text="#Settings_Safeguards_OpenChat" convar="mom_run_safeguard_chat_open" tags="chat">
+				<SettingsEnumDropDown text="#Settings_Safeguards_SwitchTracks" convar="mom_run_safeguard_switch_tracks" tags="tracks">
 					<Label id="colorize1" text="#Settings_Safeguards_ModesNone" value="0" />
 					<Label id="colorize2" text="#Settings_Safeguards_Modes1" value="1" />
 					<Label id="colorize3" text="#Settings_Safeguards_Modes2" value="2" />
 				</SettingsEnumDropDown>
 
-				<SettingsEnumDropDown text="#Settings_Safeguards_RestartStage" convar="mom_run_safeguard_restart_stage" tags="stage">
+				<SettingsEnumDropDown text="#Settings_Safeguards_SavestateTele" convar="mom_run_safeguard_saveloc_tele" tags="teleport">
+					<Label id="colorize1" text="#Settings_Safeguards_ModesNone" value="0" />
+					<Label id="colorize2" text="#Settings_Safeguards_Modes1" value="1" />
+					<Label id="colorize3" text="#Settings_Safeguards_Modes2" value="2" />
+				</SettingsEnumDropDown>
+
+				<SettingsEnumDropDown text="#Settings_Safeguards_OpenChat" convar="mom_run_safeguard_chat_open" tags="chat">
```

Comment:
Do these actually work?

Reply:
Not yet, we still have to hook safeguards up. I will probably get them working this week.